Sources/Observers/KeyboardObserver.swift

```diff
@@ -61,7 +65,17 @@ public final class KeyboardObserver: NSObject {
 }
 
 extension KeyboardObserver: UIGestureRecognizerDelegate {
-    public func gestureRecognizerShouldBegin(_ gestureRecognizer: UIGestureRecognizer) -> Bool {
-        return isKeyboardVisible
+
+    public func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer, shouldReceive touch: UITouch) -> Bool {
+
+        guard isKeyboardVisible, let view = touch.view else { return false }
+
+        for cls in ignoredViews {
```

Comment:
maybe `return ignoredViews.contains { view.isKind(of: $0) } == false` ? 💄

Reply:
Done. 😂